include/ultrasonic.h

```diff
@@ -0,0 +1,30 @@
+#include <stdint.h>
+#include "timing.h"
+
+
+#ifndef _ULTRASONIC_H_
```

Comment:
inclusion guard should be before any other code (this wouldn't cause a problem but it is good practice)